pkg/agent/core/ngt/service/kvs/kvs_test.go

```diff
@@ -121,62 +123,140 @@ func Test_bidi_Get(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           key: "",
-		       },
-		       fields: fields {
-		           ou: nil,
-		           uo: nil,
-		           l: 0,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           key: "",
-		           },
-		           fields: fields {
-		           ou: nil,
-		           uo: nil,
-		           l: 0,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		func() test {
+			fields := fields{
+				l: 0,
+			}
+			for i := 0; i < slen; i++ {
+				fields.ou[i] = new(ou)
+				fields.uo[i] = new(uo)
+			}
+
+			var (
+				key        = "45637ec4-c85f-11ea-87d0"
+				val uint32 = 14438
+			)
+
+			return test{
+				name: "return the value when there is a value for the key",
+				args: args{
+					key: key,
+				},
+				fields: fields,
+				beforeFunc: func(a args, bm BidiMap) {
+					bm.Set(a.key, val)
+				},
+				want: want{
+					want:  val,
+					want1: true,
+				},
+			}
+		}(),
+		func() test {
+			fields := fields{
+				l: 0,
+			}
+			for i := 0; i < slen; i++ {
+				fields.ou[i] = new(ou)
+				fields.uo[i] = new(uo)
+			}
+
+			var (
+				key1        = "45637ec4-c85f-11ea-87d0"
+				val1 uint32 = 14438
+			)
+			var key2 = "84a333-59633fd4-4553-414a"
```

Comment:
could you use a more meaningful name?

Reply:
thank you.
I fixed: https://github.com/vdaas/vald/pull/1223/commits/e660f2987b3ce4f611749b9d444445cfd10acc11

I deleted the variables that are not needed in the first place. And I added a meaningful variable name.:pray: